github_discovery.py

```diff
@@ -1072,6 +1117,7 @@ def process_products(data):
     log.critical('Unable to connect to Slack.')
     raise SystemExit(e) from e
 
+  alertmanager_json_data = get_alertmanager_data()
```

Comment:
I'd put this line in the `# Main loop` - that way we will get fresh alertmanager data every time it runs - every 2 hours.  With it where it is the data will only get refresh when the pod restarts  - which might be days